modules/local/PEPPAN/main.nf

```diff
@@ -0,0 +1,48 @@
+process PEPPAN {
+    tag "$meta.id"
+    label 'process_medium'
+    conda '$moduleDir/environment.yml'
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://cidgoh/peppan:latest' :
+        'cidgoh/peppan:latest' }"
+
+    input:
+    tuple val(meta), path(gff)
+    path (reference_gff)
```

Comment:
`path reference_gff`

Reply:
Sorry, I'm not sure I follow this one. Do you mean you would like the line to be like this instead:
`tuple val(meta), path(gff), path(reference_gff)`

Reply:
sorry for the confusion. No I was just referring to changing line 11 to `path reference_gff` i-e removing the parenthesis around the variable. It doesn't change anything but I realized that it is inconsistent across the workflow. I will go through this across modules once we have merged these changes.